README.md

```diff
@@ -18,6 +19,30 @@ don't always test older versions.
 If you're interested in using the [GraphQL API v4][], the recommended library is
 [shurcooL/githubv4][].
 
+## Installation ##
+
+go-github is compatible with modern Go releases in module mode, with Go installed:
+
+```bash
+go get -u github.com/google/go-github/v33
```

Comment:
We should not recommend the `-u` flag in module mode. It was needed in GOPATH mode because there was no other mechanism to ensure compatible versions of dependencies would be used, but in module mode the go.mod file provides well-tested minimum dependency version requirements.

```suggestion
go get github.com/google/go-github/v33
```